common/reviews/api/rush-lib.api.md

```diff
@@ -220,6 +223,16 @@ export class FileSystemBuildCacheProvider {
 // @beta
 export type GetCacheEntryIdFunction = (options: IGenerateCacheEntryIdOptions) => string;
 
+// @beta (undocumented)
+export interface IBuildTimeRecord {
```

Comment:
To clarify that these interfaces are related to telemetry, maybe we could add prefixes:

- `IBuildTimeRecord` --> `ITelemetryBuildTimeRecord`
- `IDependencyGraph` --> `ITelemetryDependencyGraph`
- `IDependencyGraphEntry` --> `ITelemetryDependencyGraphEntry`
- `IMachineInfo` --> `ITelemetryMachineInfo`
- . . .